src/Avalonia.Base/Media/StreamGeometryContext.cs

```diff
@@ -128,5 +128,20 @@ public void Dispose()
         {
             _impl.Dispose();
         }
+
+        /// <summary>
+        /// Draws a line to the specified point.
+        /// </summary>
+        /// <param name="point">The destination point.</param>
+        /// <param name="isStroked">Whether the segment is stroked</param>
+        public void LineTo(Point point, bool isStroked)
```

Comment:
"hasStroke", "drawStroke" or "includeStroke" might be better names for the parameter. I do prefer "Is" when naming bool; however, "stroked" isn't a standard word. "isStroked" has good symmetry with "isFilled" though... so just ideas.

Reply:
The name was taken from wpf's, but if there's better wording for it, I could change it.

Reply:
Ah ok, if naming is coming over from WPF its fine and I don't have concerns other than noted below (line should always have a stroke I think).